pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -36,24 +36,63 @@ func TestREST(t *testing.T) {
 }
 
 func TestRESTCreate(t *testing.T) {
-	summary := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
-	}
-	expectedObj := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
+	addedTime := metav1.Now()
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		nodeName    string
+		expectedObj runtime.Object
+		err         error
+		timeAdded   bool
```

Comment:
the name is a bit confusion IMO. `timeAdded` to me means that the time is added by the `Create` function, but that's not how you use it. I would suggest something less ambiguous, such as `timestampMissing`.